corehq/apps/reports/v2/models.py

```diff
@@ -46,6 +51,16 @@ def get_data_endpoint(self, endpoint_slug):
     def get_options_endpoint(self, endpoint_slug):
         return self._get_endpoint(endpoint_slug, self.options_endpoints)
 
+    def get_column_filter(self, filter_type):
+        name_to_class = dict([(f.filter_type, f) for f in self.column_filters])
```

Comment:
```suggestion
        name_to_class = {f.filter_type: f for f in self.column_filters}
```

Reply:
or for that matter, this could also be a `for` loop that short circuits when it finds one matching the `filter_type`

Reply:
updated in new PR https://github.com/dimagi/commcare-hq/pull/24099 😄